src/test/scala/kinesis/mock/cache/CacheConfigTests.scala

```diff
@@ -139,15 +139,8 @@ class CacheConfigTests
         val res = List(
           CacheConfig
             .initializeStreamsReader(awsRegion, s":$streamName1"),
-          CacheConfig
-            .initializeStreamsReader(awsRegion, s"$streamName1:"),
           CacheConfig
             .initializeStreamsReader(awsRegion, s",$streamName1:3"),
-          CacheConfig
```

Comment:
Can you add a test that asserts that providing no shard count works properly?

Reply:
Good point, thanks!
I added the tests in the latest commit (there was actually still a bug when no shardCount was given at all).